src/app/pages/fund/fund.tsx

```diff
@@ -18,37 +16,26 @@ export function FundPage() {
   const stxCryptoCurrencyAssetBalance = useStxCryptoCurrencyAssetBalance();
   const { currency } = useParams();
 
-  function getSymbol() {
-    if (isCryptoCurrency(currency)) {
-      return currency;
-    }
-    return 'STX';
-  }
-  function getAddress() {
-    switch (symbol) {
-      case 'BTC':
-        return bitcoinSigner?.address;
-      case 'STX':
-        return currentStxAccount?.address;
-    }
-  }
-  function getBalance() {
-    switch (symbol) {
-      case 'BTC':
-        return btcCryptoCurrencyAssetBalance;
-      case 'STX':
-        return stxCryptoCurrencyAssetBalance;
-    }
-  }
-
-  const symbol = getSymbol();
-  const address = getAddress();
-  const balance = getBalance();
+  const { address, balance, blockchain, symbol } =
+    currency === 'BTC'
+      ? {
+          address: bitcoinSigner?.address,
+          balance: btcCryptoCurrencyAssetBalance,
+          blockchain: 'Bitcoin',
+          symbol: currency,
+        }
+      : {
+          address: currentStxAccount?.address,
+          balance: stxCryptoCurrencyAssetBalance,
+          blockchain: 'Stacks',
+          symbol: 'STX',
+        };
```

Comment:
Also not fan of ternary here. This is a less flexible implementation than before. 

There _just happens_ to be two options of currency here, but it isn't a boolean-like data type. We could have more in future, and an implementation that assumes there will only ever be two is problematic.

All the `whenNetwork` etc helpers exist for this reason.

Reply:
Could use a map here instead too